### DISCUSSION TITLE
ESP32-CAM startet dauernd - Image with size 0 loaded --> reboot to be done!

### DISCUSSION THREAD

**Discussion by efgiese** [jomjol/AI-on-the-edge-device#2539] [Q&A]
Hallo zusammen,

ich komme mit dem Setup nicht weiter. Das Modul startet alle 5 s neu. Ich habe schon eine neue angebaut, aber mit der wird es auch nicht besser. Eigentlich wird die Kamera erkannt, aber anscheinend wird kein Bild erzeugt.

Hier ist mein Log:
`
Logs
ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:7664
load:0x40078000,len:15360
load:0x40080400,len:3744
entry 0x4008067c
I (27) boot: ESP-IDF 4.3.2 2nd stage bootloader
I (27) boot: compile time 20:02:47
I (27) boot: chip revision: 3
I (30) boot_comm: chip revision: 3, min. bootloader chip revision: 0
I (37) boot.esp32: SPI Speed      : 40MHz
I (42) boot.esp32: SPI Mode       : DIO
I (46) boot.esp32: SPI Flash Size : 4MB
I (51) boot: Enabling RNG early entropy source...
I (56) boot: Partition Table:
I (60) boot: ## Label            Usage          Type ST Offset   Length
I (67) boot:  0 nvs              WiFi data        01 02 00009000 00004000
I (75) boot:  1 otadata          OTA data         01 00 0000d000 00002000
I (82) boot:  2 phy_init         RF data          01 01 0000f000 00001000
I (89) boot:  3 ota_0            OTA app          00 10 00010000 001db000
I (97) boot:  4 ota_1            OTA app          00 11 001f0000 001db000
I (104) boot: End of partition table
I (109) boot_comm: chip revision: 3, min. application chip revision: 0
I (116) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=46fbch (290748) map
I (236) esp_image: segment 1: paddr=00056fe4 vaddr=3ffb0000 size=03ce0h ( 15584) load
I (242) esp_image: segment 2: paddr=0005accc vaddr=40080000 size=0534ch ( 21324) load
I (252) esp_image: segment 3: paddr=00060020 vaddr=400d0020 size=1575fch (1406460) map
I (790) esp_image: segment 4: paddr=001b7624 vaddr=4008534c size=15208h ( 86536) load
I (841) boot: Loaded app from partition at offset 0x10000
I (841) boot: Disabling RNG early entropy source...
I (853) quad_psram: This chip is ESP32-D0WD
I (853) esp_psram: Found 8MB PSRAM device
I (853) esp_psram: Speed: 40MHz
I (853) esp_psram: PSRAM initialized, cache is in low/high (2-core) mode.
W (860) esp_psram: Virtual address not enough for PSRAM, map as much as we can. 4MB is mapped
I (869) cpu_start: Pro cpu up.
I (871) cpu_start: Starting app cpu, entry point is 0x40082e48
I (0) cpu_start: App cpu up.
I (1765) esp_psram: SPI SRAM memory test OK
I (1774) cpu_start: Pro cpu start user code
I (1774) cpu_start: cpu freq: 160000000 Hz
I (1774) cpu_start: Application information:
I (1775) cpu_start: Project name:     AI-on-the-edge
I (1779) cpu_start: App version:      v15.3.0
I (1784) cpu_start: Compile time:     Jul 22 2023 09:43:02
I (1789) cpu_start: ELF file SHA256:  29d38f3fa1a9857c...
I (1794) cpu_start: ESP-IDF:          5.0.2
I (1798) cpu_start: Min chip rev:     v0.0
I (1802) cpu_start: Max chip rev:     v3.99 
I (1806) cpu_start: Chip rev:         v3.0
I (1810) heap_init: Initializing. RAM available for dynamic allocation:
I (1816) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
I (1821) heap_init: At 3FFBD890 len 00022770 (137 KiB): DRAM
I (1826) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
I (1832) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
I (1837) heap_init: At 4009A554 len 00005AAC (22 KiB): IRAM
I (1843) esp_psram: Adding pool of 4083K of PSRAM memory to heap allocator
I (1851) spi_flash: detected chip: generic
I (1853) spi_flash: flash io: dio
I (1868) cpu_start: Starting scheduler on PRO CPU.
I (0) cpu_start: Starting scheduler on APP CPU.
I (1869) esp_psram: Reserving pool of 40K of internal memory for DMA/internal allocations
I (1869) MAIN: 

================ Start app_main =================
I (1969) MAIN: =================================================
I (1979) MAIN: ==================== Start ======================
I (1999) MAIN: =================================================
I (2009) MAIN: PSRAM size: 8388608 byte (8MB / 64MBit)
I (2019) MAIN: Total heap: 4376759 byte
I (2039) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (4039) gpio: GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2 
I (4039) cam_hal: cam init ok
I (4039) sccb: pin_sda 26 pin_scl 27
I (4039) sccb: sccb_i2c_port=1

I (4039) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (4079) camera: Detected camera at address=0x30
I (4079) camera: Detected OV2640 camera
I (4079) camera: Camera PID=0x26 VER=0x42 MIDL=0x7f MIDH=0xa2
I (4169) cam_hal: buffer_size: 32768, half_buffer_size: 4096, node_buffer_size: 2048, node_cnt: 16, total_cnt: 15
I (4169) cam_hal: Allocating 61440 Byte frame buffer in PSRAM
I (4169) cam_hal: cam config ok
I (4169) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (6249) MAIN: Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
I (6259) SDCARD: Basic R/W check started...
I (6319) SDCARD: Basic R/W check successful
I (6379) SNTP: TimeServer not defined, using default: pool.ntp.org
I (6399) SNTP: Configuring NTP Client...
I (6409) SNTP: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
I (6419) SNTP: The local time is unknown, starting with 1970-01-01 01:00:04
I (6439) SNTP: Once the NTP server provides a time, we will switch to that one
I (6459) MAIN: CPU frequency: 160 MHz
I (6479) OTA: Start CheckOTAUpdateCheck...
I (6489) OTA: SHA-256 for the partition table: : 9d4a2809ae90fb20709a8ca3dd3ac3462e1e73286bcd31b689060c91de06fa3a
I (6509) OTA: SHA-256 for bootloader: : 4a9573dafac5d5c79c43e5a356568332da3a99dd2e220da6ceeadc96d32708a2
I (6989) OTA: SHA-256 for current firmware: : b64a22854eb1e33763bde88abf63d86bb137038b818f619734b7e59e5fc8e6b6
I (7029) SDCARD: Folder/file presence check started...
I (7059) SDCARD: Folder/file presence check successful
I (7079) MAIN: Tag: 'v15.3.0', Release: v15.3.0 (Commit: 3fbff0a), Date/Time: 2023-07-22 09:42, Web UI: Release: v15.3.0 (Commit: 3fbff0a)
I (7119) MAIN: Reset reason: Power-on event (or reset button)
I (7139) WLANINI: SSID: giese
I (7149) WLANINI: Password: XXXXXXXX
I (7159) WLANINI: Hostname: watermeter
I (7179) WLANINI: RSSIThreshold: -75
I (7189) MAIN: WLAN config loaded, init WIFI...
I (7209) WIFI: Automatic interface config --> Use DHCP service
I (7229) wifi:wifi driver task: 3ffd119c, prio:23, stack:6144, core=0
I (7229) system_api: Base MAC address is not set
I (7229) system_api: read default base MAC address from EFUSE
I (7239) wifi:wifi firmware version: 57982fe
I (7249) wifi:wifi certification version: v7.0
I (7249) wifi:config NVS flash: enabled
I (7249) wifi:config nano formating: enabled
I (7249) wifi:Init data frame dynamic rx buffer num: 16
I (7249) wifi:Init management frame dynamic rx buffer num: 16
I (7259) wifi:Init management short buffer num: 32
I (7259) wifi:Init static tx buffer num: 16
I (7269) wifi:Init tx cache buffer num: 16
I (7269) wifi:Init static rx buffer size: 1600
I (7279) wifi:Init static rx buffer num: 16
I (7279) wifi:Init dynamic rx buffer num: 16
I (7289) wifi_init: rx ba win: 16
I (7289) wifi_init: tcpip mbox: 32
I (7289) wifi_init: udp mbox: 6
I (7289) wifi_init: tcp mbox: 6
I (7299) wifi_init: tcp tx win: 5744
I (7299) wifi_init: tcp rx win: 5744
I (7299) wifi_init: tcp mss: 1440
I (7309) wifi_init: WiFi/LWIP prefer SPIRAM
I (7309) wifi_init: WiFi IRAM OP enabled
I (7309) wifi_init: WiFi RX IRAM OP enabled
I (7339) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07
I (7439) wifi:mode : sta (a8:42:e3:55:b1:c4)
I (7439) wifi:enable tsf
I (7449) WIFI: Set hostname to: watermeter
I (7459) WIFI: Init successful
I (9849) wifi:new:<6,0>, old:<1,0>, ap:<255,255>, sta:<6,0>, prof:1
I (10579) wifi:state: init -> auth (b0)
I (10589) wifi:state: auth -> assoc (0)
I (10609) wifi:state: assoc -> run (10)
I (10759) wifi:connected with giese, aid = 2, channel 6, BW20, bssid = 48:5d:35:2d:7a:2b
I (10759) wifi:security: WPA2-PSK, phy: bgn, rssi: -59
I (10769) wifi:pm start, type: 1

I (10769) WIFI: Connected to: giese, RSSI: -59
I (10789) wifi:AP's beacon interval = 102400 us, DTIM period = 1
I (11149) wifi:<ba-add>idx:0 (ifx:0, 48:5d:35:2d:7a:2b), tid:0, ssn:0, winSize:64
I (11289) wifi:<ba-del>idx
I (11309) wifi:<ba-add>idx:0 (ifx:0, 48:5d:35:2d:7a:2b), tid:0, ssn:2, winSize:64
I (12269) esp_netif_handlers: sta ip: 192.168.178.75, mask: 255.255.255.0, gw: 192.168.178.1
I (12269) WIFI: Assigned IP: 192.168.178.75
I (13119) SNTP: =================================================
I (13139) SNTP: ==================== Start ======================
I (13149) SNTP: == Logs before time sync -> log_1970-01-01.txt ==
I (13159) SNTP: Time is synced with NTP Server pool.ntp.org: 2023-07-27 18:02:52
I (13469) MAIN: Device info: CPU cores: 2, Chip revision: 300
I (13479) MAIN: SD card info: Name: 00000, Capacity: 7655MB, Free: 7633MB
I (15489) MAIN SERVER: Starting server on port: '80'
I (15489) MAIN SERVER: Registering URI handlers
I (15489) MAINCTRL: server_main_flow_task - Registering URI handlers
I (15489) OTA: Registering URI handlers
I (15489) GPIO: start GpioHandler
I (15499) GPIO: register GPIO Uri
I (15499) GPIO: server_GPIO - Registering URI handlers
I (15509) MAIN: Initialization completed successfully
I (15539) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (17909) LOGFILE: Set log level to DEBUG
I (17919) GPIO: read GPIO config and init GPIO
I (17939) GPIO: GPIO init completed, handler is disabled
I (17939) MAINCTRL: We are in Setup Mode -> Not starting Auto Flow!
E (24629) C IMG BASIS: Image with size 0 loaded --> reboot to be done! Check that your camera module is working and connected properly.
I (24689) OTA: Reboot triggered by Software (5s)
W (24699) OTA: Reboot in 5sec
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:7664
load:0x40078000,len:15360
load:0x40080400,len:3744
entry 0x4008067c
I (27) boot: ESP-IDF 4.3.2 2nd stage bootloader
I (27) boot: compile time 20:02:47
I (27) boot: chip revision: 3
I (30) boot_comm: chip revision: 3, min. bootloader chip revision: 0
I (37) boot.esp32: SPI Speed      : 40MHz
I (42) boot.esp32: SPI Mode       : DIO
I (46) boot.esp32: SPI Flash Size : 4MB
I (51) boot: Enabling RNG early entropy source...
I (56) boot: Partition Table:
I (60) boot: ## Label            Usage          Type ST Offset   Length
I (67) boot:  0 nvs              WiFi data        01 02 00009000 00004000
I (74) boot:  1 otadata          OTA data         01 00 0000d000 00002000
I (82) boot:  2 phy_init         RF data          01 01 0000f000 00001000
I (89) boot:  3 ota_0            OTA app          00 10 00010000 001db000
I (97) boot:  4 ota_1            OTA app          00 11 001f0000 001db000
I (104) boot: End of partition table
I (109) boot_comm: chip revision: 3, min. application chip revision: 0
I (116) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=46fbch (290748) map
I (236) esp_image: segment 1: paddr=00056fe4 vaddr=3ffb0000 size=03ce0h ( 15584) load
I (242) esp_image: segment 2: paddr=0005accc vaddr=40080000 size=0534ch ( 21324) load
I (252) esp_image: segment 3: paddr=00060020 vaddr=400d0020 size=1575fch (1406460) map
I (790) esp_image: segment 4: paddr=001b7624 vaddr=4008534c size=15208h ( 86536) load
I (841) boot: Loaded app from partition at offset 0x10000
I (841) boot: Disabling RNG early entropy source...
I (852) quad_psram: This chip is ESP32-D0WD
I (853) esp_psram: Found 8MB PSRAM device
I (853) esp_psram: Speed: 40MHz
I (853) esp_psram: PSRAM initialized, cache is in low/high (2-core) mode.
W (860) esp_psram: Virtual address not enough for PSRAM, map as much as we can. 4MB is mapped
I (869) cpu_start: Pro cpu up.
I (871) cpu_start: Starting app cpu, entry point is 0x40082e48
I (864) cpu_start: App cpu up.
I (1765) esp_psram: SPI SRAM memory test OK
I (1774) cpu_start: Pro cpu start user code
I (1774) cpu_start: cpu freq: 160000000 Hz
I (1774) cpu_start: Application information:
I (1775) cpu_start: Project name:     AI-on-the-edge
I (1779) cpu_start: App version:      v15.3.0
I (1783) cpu_start: Compile time:     Jul 22 2023 09:43:02
I (1789) cpu_start: ELF file SHA256:  29d38f3fa1a9857c...
I (1794) cpu_start: ESP-IDF:          5.0.2
I (1798) cpu_start: Min chip rev:     v0.0
I (1801) cpu_start: Max chip rev:     v3.99 
I (1805) cpu_start: Chip rev:         v3.0
I (1810) heap_init: Initializing. RAM available for dynamic allocation:
I (1816) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
I (1821) heap_init: At 3FFBD890 len 00022770 (137 KiB): DRAM
I (1826) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
I (1832) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
I (1837) heap_init: At 4009A554 len 00005AAC (22 KiB): IRAM
I (1843) esp_psram: Adding pool of 4083K of PSRAM memory to heap allocator
I (1851) spi_flash: detected chip: generic
I (1853) spi_flash: flash io: dio
I (1868) cpu_start: Starting scheduler on PRO CPU.
I (0) cpu_start: Starting scheduler on APP CPU.
I (1869) esp_psram: Reserving pool of 40K of internal memory for DMA/internal allocations
I (1869) MAIN: 

================ Start app_main =================
I (1969) MAIN: =================================================
I (1979) MAIN: ==================== Start ======================
I (1989) MAIN: =================================================
I (2009) MAIN: PSRAM size: 8388608 byte (8MB / 64MBit)
I (2019) MAIN: Total heap: 4376759 byte
I (2029) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (4029) gpio: GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2 
I (4029) cam_hal: cam init ok
I (4029) sccb: pin_sda 26 pin_scl 27
I (4029) sccb: sccb_i2c_port=1

I (4029) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (4069) camera: Detected camera at address=0x30
I (4069) camera: Detected OV2640 camera
I (4069) camera: Camera PID=0x26 VER=0x42 MIDL=0x7f MIDH=0xa2
I (4159) cam_hal: buffer_size: 32768, half_buffer_size: 4096, node_buffer_size: 2048, node_cnt: 16, total_cnt: 15
I (4159) cam_hal: Allocating 61440 Byte frame buffer in PSRAM
I (4159) cam_hal: cam config ok
I (4159) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (6239) MAIN: Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
I (6249) SDCARD: Basic R/W check started...
I (6299) SDCARD: Basic R/W check successful
I (6359) SNTP: TimeServer not defined, using default: pool.ntp.org
I (6369) SNTP: Configuring NTP Client...
I (6379) SNTP: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
I (6399) SNTP: Time is already set: 2023-07-27 18:03:16
I (6419) MAIN: CPU frequency: 160 MHz
I (6439) OTA: Start CheckOTAUpdateCheck...
I (6449) OTA: SHA-256 for the partition table: : 9d4a2809ae90fb20709a8ca3dd3ac3462e1e73286bcd31b689060c91de06fa3a
I (6469) OTA: SHA-256 for bootloader: : 4a9573dafac5d5c79c43e5a356568332da3a99dd2e220da6ceeadc96d32708a2
I (6949) OTA: SHA-256 for current firmware: : b64a22854eb1e33763bde88abf63d86bb137038b818f619734b7e59e5fc8e6b6
I (6989) SDCARD: Folder/file presence check started...
I (7019) SDCARD: Folder/file presence check successful
I (7039) MAIN: Tag: 'v15.3.0', Release: v15.3.0 (Commit: 3fbff0a), Date/Time: 2023-07-22 09:42, Web UI: Release: v15.3.0 (Commit: 3fbff0a)
I (7109) MAIN: Reset reason: Via esp_restart
I (7129) WLANINI: SSID: giese
I (7139) WLANINI: Password: XXXXXXXX
I (7149) WLANINI: Hostname: watermeter
I (7169) WLANINI: RSSIThreshold: -75
I (7179) MAIN: WLAN config loaded, init WIFI...
I (7189) WIFI: Automatic interface config --> Use DHCP service
I (7219) wifi:wifi driver task: 3ffd119c, prio:23, stack:6144, core=0
I (7219) system_api: Base MAC address is not set
I (7219) system_api: read default base MAC address from EFUSE
I (7229) wifi:wifi firmware version: 57982fe
I (7239) wifi:wifi certification version: v7.0
I (7239) wifi:config NVS flash: enabled
I (7239) wifi:config nano formating: enabled
I (7239) wifi:Init data frame dynamic rx buffer num: 16
I (7239) wifi:Init management frame dynamic rx buffer num: 16
I (7249) wifi:Init management short buffer num: 32
I (7249) wifi:Init static tx buffer num: 16
I (7259) wifi:Init tx cache buffer num: 16
I (7259) wifi:Init static rx buffer size: 1600
I (7269) wifi:Init static rx buffer num: 16
I (7269) wifi:Init dynamic rx buffer num: 16
I (7279) wifi_init: rx ba win: 16
I (7279) wifi_init: tcpip mbox: 32
I (7279) wifi_init: udp mbox: 6
I (7279) wifi_init: tcp mbox: 6
I (7289) wifi_init: tcp tx win: 5744
I (7289) wifi_init: tcp rx win: 5744
I (7289) wifi_init: tcp mss: 1440
I (7299) wifi_init: WiFi/LWIP prefer SPIRAM
I (7299) wifi_init: WiFi IRAM OP enabled
I (7299) wifi_init: WiFi RX IRAM OP enabled
I (7329) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07
I (7429) wifi:mode : sta (a8:42:e3:55:b1:c4)
I (7429) wifi:enable tsf
I (7429) WIFI: Set hostname to: watermeter
I (7449) WIFI: Init successful
I (9839) wifi:new:<6,0>, old:<1,0>, ap:<255,255>, sta:<6,0>, prof:1
I (10579) wifi:state: init -> auth (b0)
I (10599) wifi:state: auth -> assoc (0)
I (10619) wifi:state: assoc -> run (10)
I (10739) wifi:connected with giese, aid = 2, channel 6, BW20, bssid = 48:5d:35:2d:7a:2b
I (10739) wifi:security: WPA2-PSK, phy: bgn, rssi: -65
I (10759) wifi:pm start, type: 1

I (10759) WIFI: Connected to: giese, RSSI: -66
I (10819) wifi:AP's beacon interval = 102400 us, DTIM period = 1
I (10899) wifi:<ba-add>idx:0 (ifx:0, 48:5d:35:2d:7a:2b), tid:0, ssn:0, winSize:64
I (11759) esp_netif_handlers: sta ip: 192.168.178.75, mask: 255.255.255.0, gw: 192.168.178.1
I (11759) WIFI: Assigned IP: 192.168.178.75
I (12999) SNTP: Time is synced with NTP Server pool.ntp.org: 2023-07-27 18:03:23
I (13459) MAIN: Device info: CPU cores: 2, Chip revision: 300
I (13469) MAIN: SD card info: Name: 00000, Capacity: 7655MB, Free: 7633MB
I (15489) MAIN SERVER: Starting server on port: '80'
I (15489) MAIN SERVER: Registering URI handlers
I (15489) MAINCTRL: server_main_flow_task - Registering URI handlers
I (15489) OTA: Registering URI handlers
I (15489) GPIO: start GpioHandler
I (15499) GPIO: register GPIO Uri
I (15499) GPIO: server_GPIO - Registering URI handlers
I (15509) MAIN: Initialization completed successfully
I (15549) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (17899) LOGFILE: Set log level to DEBUG
I (17919) GPIO: read GPIO config and init GPIO
I (17929) GPIO: GPIO init completed, handler is disabled
I (17929) MAINCTRL: We are in Setup Mode -> Not starting Auto Flow!
E (24619) C IMG BASIS: Image with size 0 loaded --> reboot to be done! Check that your camera module is working and connected properly.
I (24669) OTA: Reboot triggered by Software (5s)
W (24679) OTA: Reboot in 5sec`

**Comment by friedpa** [comment]
Schau mal hier:

I (17939) MAINCTRL: We are in Setup Mode -> Not starting Auto Flow!

Da ist immer noch der Setup Modus an. Entweder in der config.ini ändern über das WebInterface, oderr wenn das nicht geht über ein Edit auf der SD Card selbst.

**Comment by efgiese** [comment]
Ich bin nicht sicher, ob das zielführend ist. Das Setup ist ja auch noch gar nicht gelaufen. Auf der Website wir auch kein Bild angezeigt.--Mit freundlichen GrüßenEdgar GieseAm 27.07.23, 18:59 schrieb Paul Fried ***@***.***>:

   
  Schau mal hier: 
  I (17939) MAINCTRL: We are in Setup Mode -> Not starting Auto Flow! 
  Da ist immer noch der Setup Modus an. Entweder in der config.ini ändern über das WebInterface, oderr wenn das nicht geht über ein Edit auf der SD Card selbst.

**Comment by friedpa** [comment]
Ok, dachte der Setup ist schon durch....
Dann scheint es die nächste Zeile zu sein:

E (24619) C IMG BASIS: Image with size 0 loaded --> reboot to be done! Check that your camera module is working and connected properly.

Die Camera liefert kein Bild.

**Comment by efgiese** [comment]
Genau das ist das Problem. Fehlt nur noch die Lösung...

Es sieht aber so aus, als wenn die Kamera richtig erkannt wird. Also, was geht dann schief. Kann es ein Problem beim Flashen gegeben haben? Die Software scheint aber auch installiert zu sein...

I (4079) camera: Detected camera at address=0x30
I (4079) camera: Detected OV2640 camera
I (4079) camera: Camera PID=0x26 VER=0x42 MIDL=0x7f MIDH=0xa2
I (4169) cam_hal: buffer_size: 32768, half_buffer_size: 4096, node_buffer_size: 2048, node_cnt: 16, total_cnt: 15
I (4169) cam_hal: Allocating 61440 Byte frame buffer in PSRAM
I (4169) cam_hal: cam config ok
I (4169) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (6249) MAIN: Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2

**Comment by friedpa** [comment]
Meiner Meinung nach zwei mögliche Gründe: die Camera liefert kein Bild, oder die Daten können nicht auf die SD Card geschrieben werden. Vielleicht kann aber hier @caco3 besser Auskunft geben. Ich würde einmal probieren, ob es an der SD Card liegt und eine andere SD Card versuchen.

**Reply by efgiese** [reply]
Ich habe schon verschiedene SD Karten ausprobiert. Auch eine andere Kamera zeigt das gleiche Problem.
Ich Versuche mal ein anderes Modul...

**Comment by Slider0007** [comment]
Bitte poste mal deine config.ini

**Reply by efgiese** [reply]
Die config.ini habe ich gerade nicht zur Hand. Ich habe allerdings nichts, außer Loglevel=4, geändert und nutze die 15.3.0.

**Reply by Slider0007** [reply]
Prüfe bitte die [TakeImage] Sektion, da sollte nichts außer maximal den Image Logging Parametern mit `;` vor den Zeilen auskommentiert sein.

Beispiel:

```
[TakeImage]
;RawImagesLocation = /log/source
;RawImagesRetention = 5
WaitBeforeTakingPicture = 0.5
ImageQuality = 10
ImageSize = VGA
LEDIntensity = 4
Brightness = -2
Contrast = 1
Saturation = 0
FixedExposure = false
Demo = false
```

**Comment by efgiese** [comment]
so hier ist nun meine config.ini

```
[TakeImage]
;RawImagesLocation = /log/source
WaitBeforeTakingPicture = 5
;RawImagesRetention = 15
Brightness = 0
Contrast = 0
Saturation = 0
LEDIntensity = 50
ImageQuality = 12
ImageSize = VGA
FixedExposure = false
Demo = false

[Alignment]
InitialRotate = 0.0
InitialMirror = false
SearchFieldX = 20
SearchFieldY = 20
AlignmentAlgo = Default
FlipImageSize = false
/config/ref0.jpg 103 271
/config/ref1.jpg 442 142

[Digits]
Model = /config/dig-cont_0620_s3_q.tflite
CNNGoodThreshold = 0.5
;ROIImagesLocation = /log/digit
;ROIImagesRetention = 3
main.dig1 294 126 30 54 false
main.dig2 343 126 30 54 false
main.dig3 391 126 30 54 false

[Analog]
Model = /config/ana-cont_1207_s2_q.tflite
CNNGoodThreshold = 0.5
;ROIImagesLocation = /log/analog
;ROIImagesRetention = 3
main.ana1 432 230 92 92 false
main.ana2 379 332 92 92 false
main.ana3 283 374 92 92 false
main.ana4 155 328 92 92 false

[PostProcessing]
main.DecimalShift = 0
main.AnalogDigitalTransitionStart = 9.2
PreValueUse = true
PreValueAgeStartup = 720
main.AllowNegativeRates = false
main.MaxRateValue = 0.05
;main.MaxRateType = AbsoluteChange
main.ExtendedResolution = false
main.IgnoreLeadingNaN = false
ErrorMessage = true
CheckDigitIncreaseConsistency = false

;[MQTT]
;Uri = mqtt://IP-ADRESS:1883
;MainTopic = watermeter
;ClientID = watermeter
;user = USERNAME
;password = PASSWORD
RetainMessages = false
HomeassistantDiscovery = false
;MeterType = other

;[InfluxDB]
;Uri = undefined
;Database = undefined
;Measurement = undefined
;user = undefined
;password = undefined

;[InfluxDBv2]
;Uri = undefined
;Database = undefined
;Measurement = undefined
;Org = undefined
;Token = undefined
main.Field = undefined

;[GPIO]
;MainTopicMQTT = wasserzaehler/GPIO
;IO0 = input disabled 10 false false 
;IO1 = input disabled 10 false false 
;IO3 = input disabled 10 false false 
;IO4 = built-in-led disabled 10 false false 
;IO12 = input-pullup disabled 10 false false 
;IO13 = input-pullup disabled 10 false false
LEDType = WS2812
LEDNumbers = 2
LEDColor = 150 150 150 

[AutoTimer]
AutoStart = true
Interval = 5

[DataLogging]
DataLogActive = true
DataFilesRetention = 3

[Debug]
LogLevel = 4
LogfilesRetention = 3

[System]
TimeZone = CET-1CEST,M3.5.0,M10.5.0/3
;TimeServer = pool.ntp.org
;Hostname = undefined
RSSIThreshold = -75
CPUFrequency = 160
SetupMode = true
```

**Reply by Slider0007** [reply]
OK danke, leider nicht zu erkennen. War nur eine Vermutung.

**Comment by efgiese** [comment]
Vielen Dank  für Eure Mühe.

Ich habe es mit dem Modul aufgegeben und schicke es zurück.

Bye...
